easybuild/easyconfigs/t/tmux/tmux-2.3.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'tmux'
+version = '2.3'
+
+homepage = 'http://tmux.sourceforge.net/'
+description = """tmux is a terminal multiplexer. It lets you switch easily 
+between several programs in one terminal, detach them (they keep 
+running in the background) and reattach them to a different terminal."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
@foxik0070 You need to use an empty version (`''`) to ensure that the specified dependencies are loaded, see also http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#loading-of-modules-for-dependencies-with-a-dummy-toolchain .

This explains why my test report includes a failure for `tmux`.

In the test report you shared, you had `libevent` loaded manually, which is sort of cheating. ;-)